fastlane/Fastfile

```diff
@@ -420,27 +420,29 @@ platform :android do
     create_backmerge_pr
   end
 
-  # This lane publishes a release on GitHub and creates a PR to backmerge the current release branch into the next release/ branch
+  # This lane publishes a release on GitHub. If `track` is `production`, it will also delete the release branch.
   #
   # @param [Boolean] skip_confirm (default: false) If set, will skip the confirmation prompt before running the rest of the lane
+  # @param [String] track (default: 'production') The Google Play track the release was published to in Google Play Console. Must be either `beta` or `production`.
+  #        If 'production': The GitHub Release will be expected to have a title like `X.Y`, and this lane will also delete the release branch after publishing the GitHub Release.
+  #        If 'beta': The draft GitHub Release will be expected to have a title like `X.Y-rc-N`. The release branch will NOT be deleted for those.
   #
-  lane :publish_release do |skip_confirm: false|
+  lane :publish_gh_release do |skip_confirm: false, track: 'production'|
```

Comment:
ℹ️ Note: I had to rename this lane from `publish_github_release` (the name I chose to rename it to in the first iterations of that PR) to `publish_gh_release` because Fastlane doesn't allow you to have a lane have the same lane as an existing action (and indeed we have a fastlane action already using the name `publish_github_release`… and in fact that's the action that this lane calls on line 450)